
try{
    let car = null;
}
catch(error){
    console.log('error: ', error);
}
finally{
    console.log('this always executes');
}
